modules/core/src/v2/internal/mpcUtils.ts

```diff
@@ -14,6 +21,85 @@ export abstract class MpcUtils {
     this.baseCoin = baseCoin;
   }
 
+  protected async getBitgoGpgPubKey(): Promise<Key> {
+    const constants = await this.bitgo.fetchConstants();
+    if (!constants.mpc || !constants.mpc.bitgoPublicKey) {
```

Comment:
https://github.com/BitGo/bitgo-microservices/blob/develop/packages/wallet-platform/app/controllers/api/client.js#L45

Wallet platform returns this as constants.tss not constants.mpc

Reply:
Right, we are changing it in this PR https://github.com/BitGo/bitgo-microservices/pull/15616/files#diff-35b5736f90ff9fc9818876d37c5df36f7ff8a0589d7a87ec96bac15192e349c8R44